roles/libvirt_manager/tasks/clean_layout.yml

```diff
@@ -154,3 +154,8 @@
         - workload
         - images
         - volumes
+
+    - name: Clean virtualBMC
+      ansible.builtin.import_role:
+        name: virtualbmc
```

Comment:
why we stopped using FQCN?

Reply:
`ansible.builtin.import_role` looks like FQCN to me?

Reply:
ok, so imported/included role may also have an FQCN - that's new to me, and to the framework actually - NONE of the calls have FQCN pointing to our collection. So it's more "why don't we start using FQCN".

I propose to merge as-is and get someone to work on updating the calls - or consider using `collections` in the playbooks, though it's hiding the thing IMHO.